src/compile/common.ts

```diff
@@ -130,65 +127,14 @@ export function formatMixins(model: Model, channel: Channel, format: string) {
         def.format = model.config().numberFormat;
         break;
       case TEMPORAL:
-        def.format = timeFormat(model, channel) || model.config().timeFormat;
+        def.format = timeFormatExpr(fieldDef.timeUnit, shortTimeLabels) || model.config().timeFormat;
```

Comment:
We do that all the time unless you need to make `timeFormat` a local variable, which isn't true in this case :)